app/controllers/application_controller.rb

```diff
@@ -1,12 +1,13 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  helper_method :current_user, :client
```

Comment:
very cool, I've never seen this before!
you use current_user in the user_drops/index, right? Where do you use client?